Workflow creating .pot file fails because of restrictions on main branch

```plaintext
Run git config --global user.name "GitHub Actions"
[main b3345da] Update reuse.pot
 1 file changed, 43 insertions(+), 36 deletions(-)
remote: error: GH006: Protected branch update failed for refs/heads/main.        
remote: error: Changes must be made through a pull request.        
To https://github.com/fsfe/reuse-tool
 ! [remote rejected] main -> main (protected branch hook declined)
error: failed to push some refs to 'https://github.com/fsfe/reuse-tool'
Error: Process completed with exit code 1.
```

--- a/.github/workflows/gettext.yaml
+++ b/.github/workflows/gettext.yaml
@@ -15,9 +15,13 @@ jobs:
   create-pot:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 1
+          # The main branch is protected. fsfe-system has been granted an
+          # exception to the branch protection, so we'll use that account's
+          # token to push to the main branch.
+          token: ${{ secrets.FSFE_SYSTEM_TOKEN }}
       - name: Install gettext
         run: sudo apt-get install -y gettext
       - name: Create .pot file
@@ -32,7 +36,7 @@ jobs:
       - name: Commit and push updated reuse.pot
         if: ${{ steps.diff.outputs.changed != '0' }}
         run: |
-          git config --global user.name "GitHub Actions"
+          git config --global user.name "fsfe-system"
           git config --global user.email "<>"
           git add po/reuse.pot
           git commit -m "Update reuse.pot"

